### PR TITLE
add phone, email and address to patient registration

### PR DESCRIPTION
**Pull request by Andati**

### CODE REVIEW

**Comment by Andati** [comment]
TODO: Figure out how resource mapper will parse:-
{
    "linkId": "PR-telecom-value",
    "definition": "http://hl7.org/fhir/StructureDefinition/Patient#Patient.telecom.system",
    "type": "string",
    "text": "phone"
  }

Also how to hide this from displaying in UI

File: app/src/main/assets/patient-registration.json [hunk 1]
```diff
@@ -62,6 +62,76 @@
...
+          ],
+          "item": [
+            {
+              "linkId": "PR-telecom-system",
+              "definition": "http://hl7.org/fhir/StructureDefinition/Patient#Patient.telecom.system",
+              "type": "string",
+              "text": "system",
+              "initial": [
+                {
+                  "valueString": "phone"
```
Comment (pld) [inline app/src/main/assets/patient-registration.json:87]: can we force this to be numeric?

related to https://github.com/OpenSRP/fhircore/issues/46

**Review by pld — APPROVED** [review]

File: app/src/main/assets/patient-registration.json (hunk 1 above)
Comment (Andati) [reply app/src/main/assets/patient-registration.json:87]: So the function here expects a string https://github.com/hapifhir/org.hl7.fhir.core/blob/e69bad679afdc8382dd309f7341f1c43d96230dc/org.hl7.fhir.r4/src/main/java/org/hl7/fhir/r4/model/ContactPoint.java#L491

Comment (Andati) [reply app/src/main/assets/patient-registration.json:87]: It's possible to extend but let me wait for reviews here https://github.com/google/android-fhir/pull/326

**Comment by pld** [comment]
Yes if you want to allow formatting and country codes can see why you'd want a string, let's merge this in but note that we'll need to investigate enforcing eg regex checks for questionnaire fields

**Pull request merged**